controllers/metrics.go

```diff
@@ -0,0 +1,196 @@
+package controllers
+
+import (
+	"context"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/common/log"
+	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+var (
+	// 'backupPhase' - [New, FailedValidation, InProgress, Uploading, UploadingPartialFailure, Completed, PartiallyFailed, Failed, Deleting]
+	// 'restorePhase' - [New, FailedValidation, InProgress, Completed, PartiallyFailed, Failed]
+	backupGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "oadp_app_workload_backup_total",
```

Comment:
if it's a gauge, it shouldn't have the `_total` suffix (it's reserved for counters).